lib/ansible/modules/cloud/vmware/vmware_content_library_info.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_content_library_info
+short_description: Gather information about VMWare Content Library
+description:
+- Module to list the content libraries.
+- Module to get facts about specific content library
+- Content Library feature is introduced in vSphere 6.0 version, so this module is not supported in the earlier versions of vSphere.
+- All variables and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Pavan Bidkar (@pgbidkar)
+notes:
+- Tested on vSphere 6.5, 6.7
+requirements:
+- python >= 2.6
+- PyVmomi
+- vSphere Automation SDK
+options:
+    library_id:
+      description:
+      - content library id for which details needs to be fetched
+      type: str
+      required: False
+extends_documentation_fragment: vmware_rest_client.documentation
+'''
+
+EXAMPLES = r'''
+- name: Get List of Content Libraries
+  vmware_content_library_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+  delegate_to: localhost
+
+- name: Get facts about content library
+  vmware_content_library_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    library_id: 'LibraryID'
```

Comment:
```suggestion
    library_id: '13b0f060-f4d3-4f84-b61f-0fe1b0c0a5a8'
```
This will hint user that, user is expected to enter something similar.